SCSH_3C_iOS/SCSH_3C_iOS/Repository/Model/Product.swift

```diff
@@ -2,11 +2,22 @@
 //  Product.swift
 //  SCSH_3C_iOS
 //
-//  Created by 辜敬閎 on 2023/7/31.
+//  Created by 辜敬閎 on 2023/12/11.
```

Comment:
This is a very coooooooool change 😎

Reply:
I deleted this file on the previous commit, I added it back in this one, I've no idea why it shows like that XDD